### PR TITLE
Made changes for gaol_type not included

### PR DESCRIPTION
**Pull request by palghadmalavinash**
Changes made to add new goal_types and additional parameters that are required for this goal_types.

### CODE REVIEW

File: terminalone/models/campaign.py [hunk 1]
```diff
@@ -76,6 +76,9 @@ class Campaign(Entity):
         'use_mm_freq': t1types.int_to_bool,
         'version': int,
         'zone_name': None,
+        'viewability_type': str,
```
Comment (FodT) [inline terminalone/models/campaign.py]: Strings have a type of None in the `_pull` section

File: terminalone/models/campaign.py [hunk 2]
```diff
@@ -99,6 +102,9 @@ class Campaign(Entity):
         'status': int,
         'use_default_ad_server': int,
         'use_mm_freq': int,
+        'viewability_type': str,
+        'viewability_vendor_id': int,
+        'viewability_sample_rate': float
```
Comment (FodT) [inline terminalone/models/campaign.py]: You only need to declare the push types if they're different from `_pull` (ie, need to be converted) as `_push` starts with a copy.

**Comment by FodT** [comment]
Hi @palghadmalavinash - thanks for this PR! I've made a few comments on the code. Also, the travis build is failing on unit tests - I can't merge until the issues are resolved :)

**Comment by palghadmalavinash** [comment]
Hi @FodT, Thanks for the comments, Have made the changes accordingly. Can you please have look.
Thanks and Regards.
Avinash

**Comment by swparekh** [comment]
Hi @FodT if the changes look good can we merge this today?

**Pull request merged**